src/Evolution/Systems/CurvedScalarWave/Equations.hpp

```diff
@@ -0,0 +1,110 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/GrTags.hpp"
+#include "Utilities/TMPL.hpp"
+
+template <typename>
+class Variables;
+
+class DataVector;
+
+namespace gsl {
+template <typename T>
+class not_null;
```

Comment:
I don't think a forward declaration is sufficient here.

Reply:
Yea, I don't think it's unreasonable to include GSL, since anybody calling this function is guaranteed to need it.